addons/binding/org.openhab.binding.te923/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="te923" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- TE923 Binding -->
+	<thing-type id="weather">
+
+		<label>TE923, Irox Information</label>
+		<description>Extract information from the TE923, Irox USB devices</description>
+
+
+		<channels>
+            <!-- channel id="time" typeId="time" /-->
+            <channel id="temperature_c0" typeId="temperature_c0" />
+			<channel id="humidity_c0" typeId="humidity_c0" />
+			<channel id="temperature_c1" typeId="temperature_c1" />
```

Comment:
Are all these numbered channels for temperature/humidity always relevant? You might setting some of those to advanced so they are disabled by default.

Reply:
Did it. 

By default the internal sensor + 2 external sensors are provided. You can have up to 5 external sensor.

I have split the code in 'advanced="false"' and  'advanced="true"'.
Do you suggest a better way to do it?

		<channels>
			<!-- channel id="time" typeId="time" / -->
			<channel id="temperature_c0" typeId="temperature">
				<label>Temperature Internal Sensor</label>
				<description>Temperature in °C of the internal sensor</description>
			</channel>
			<channel id="temperature_c1" typeId="temperature">
				<label>Temperature Sensor 1</label>
				<description>Temperature in °C of the sensor on channel 1</description>
			</channel>
			<!--- some more code -->
			<channel id="temperature_c3" typeId="temperature_adv">
				<label>Temperature Sensor 3</label>
				<description>Temperature in °C of the sensor on channel 3</description>
			</channel>
			<!--- some more code -->
		</channels>


	<channel-type id="temperature" advanced="false">
		<item-type>Number</item-type>
		<label>Temperature</label>
		<description>Current temperature in degrees celsius</description>
		<category>Temperature</category>
		<state readOnly="true" pattern="%.1f °C">
		</state>
	</channel-type>


	<channel-type id="temperature_adv" advanced="true">
		<item-type>Number</item-type>
		<label>Temperature</label>
		<description>Current temperature in degrees celsius</description>
		<category>Temperature</category>
		<state readOnly="true" pattern="%.1f °C">
		</state>
	</channel-type>